elastic/apm-server
# Add ES cpu cgroup mappings

## Motivation/summary

Report correct CPU metrics from isolated processes.

closes https://github.com/elastic/apm-server/issues/4433

## Checklist




## How to test these changes



Run ES, kibana, and apm-server locally. Submit events containing the new mappings:

```
curl -XPOST http://localhost:8200/intake/v2/events \
--data-binary "@testdata/intake-v2/metricsets.ndjson" \
-H "Content-Type: application/x-ndjson" | \
jq .
```

Verify in index management that the new mappings can be seen: http://localhost:5601/app/management/data/index_management/indices
